File: 알고리즘기초1/수학과문자열/0101.cpp
#include <iostream>
using namespace std;

int main() {
    int T;
    cin >> T;
    while (T--) {
        int n;
        cin >> n;
        int cnt = 0;
        for(int i = 1; i <= n; ++i){
          if (n % i == 0) {
                ++cnt;
            }     
        }   
        if (cnt % 2 == 0){
            cout<< "Even";
        } else {
            cout<< "Odd";
        }
        cout << '\n'; 
    }

}

File: 초급6주차/알고리즘기초1/수학과문자열/0101.py
T = int(input())
for i in range(T):
    N = int(input())
    cnt = 0
    for i in range(1,N+1):
        if N % i == 0:
            cnt += 1
    if cnt % 2 == 0:
        print("Even")
    else:
        print("Odd")
